src/simply-rets-api-helper.php

```diff
@@ -534,7 +534,7 @@ public static function srDetailsGallery($listing) {
 
                     $markup .= "<img src='$image_url' "
                             . "data-title='$full_address'"
-                            . "data-description='$img_description'>";
+                            . "data-description='" . htmlentities($img_description) . "'>";
```

Comment:
This is a misc fix. Some templates process the raw HTML of a "content" block, and it doesn't like how we have HTML inside of an HTML attribute. Converting to HTML entities seems to fix the problem.

We should test this on a few themes.